reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt

```diff
@@ -38,6 +38,12 @@ class StaticHtmlReporterTest : WordSpec({
             val outputDir = createTempDir().apply { deleteOnExit() }
             StaticHtmlReporter().generateReport(ortResult, DefaultResolutionProvider(), CopyrightGarbage(), outputDir)
             outputDir.resolve("scan-report.html").isFile shouldBe true
+
+            val actualFile = outputDir.resolve("scan-report.html")
+            val expectedFile = File("src/funTest/assets/file-counter-expected-scan-report.html")
+
+            actualFile.isFile shouldBe true
```

Comment:
Line 45 is not needed imo as it does not provide additional information (compared to failing in line 46) if it fails.